test/MinCostFlow/test_mcf.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+import shutil
+from pathlib import Path
+from src.mincostflow import MinCostFlow
+from test.MEO.test_meo import OUT_FILE
```

Comment:
Can we delete this line?

Reply:
I believe so, I'm not sure why that is in there